strings/base_delegate.h

```diff
@@ -73,15 +93,16 @@ namespace winrt::impl
                 return delegate;
             }
 
+            const auto id = guid_of<T>();
```

Comment:
I don't mind either way, but it is interesting. 😊

Reply:
It seems to have made a tiny difference in folding of the resolve-and-call lambdas?  Consider these two interfaces:

```c++
[uuid(a)] interface IFoo { void Invoke(IPropertyValue v); }
[uuid(b)] interface IBar { void Invoke(IJsonValue v); }
```

The actual call sequence to call invoke is the same - load pointer-sized value of `v` from lambda storage, push value, call vtable[7]. The difference is the IID passed to `Resolve`. Making that also a member of lambda storage means this little funclet can be identical for every combination of `Invoke` parameter sets, rather than for parameter sets x interface IIDs.